tests/pyfunc/test_chat_model.py

```diff
@@ -71,7 +71,7 @@ def get_mock_response(messages, params):
     }
 
 
-class TestChatModel(mlflow.pyfunc.ChatModel):
+class SimpleChatModel(mlflow.pyfunc.ChatModel):
```

Comment:
renaming this because classes that are prefixed with `Test` are expected to be running tests, so extra logic gets run on them in pytest